homeassistant/components/notify/sendgrid.py

```diff
@@ -31,19 +34,21 @@ def get_service(hass, config, discovery_info=None):
     api_key = config.get(CONF_API_KEY)
     sender = config.get(CONF_SENDER)
     recipient = config.get(CONF_RECIPIENT)
+    sender_name = config.get(CONF_SENDER_NAME)
 
-    return SendgridNotificationService(api_key, sender, recipient)
+    return SendgridNotificationService(api_key, sender, recipient, sender_name)
 
 
 class SendgridNotificationService(BaseNotificationService):
     """Implementation the notification service for email via Sendgrid."""
 
-    def __init__(self, api_key, sender, recipient):
+    def __init__(self, api_key, sender, recipient, sender_name):
```

Comment:
Let's keep the two sender variables next to each other.

Reply:
done!